tests/CrawlerProcess/asyncio_enabled_reactor_different_loop.py

```diff
@@ -24,5 +24,6 @@ def start_requests(self):
         "ASYNCIO_EVENT_LOOP": "uvloop.Loop",
     }
 )
-process.crawl(NoRequestsSpider)
+d = process.crawl(NoRequestsSpider)
+d.addErrback(lambda failure: failure.printTraceback())
```

Comment:
I wonder if we should do it by default for CrawlerRunner, maybe with `CrawlerRunner.__init__` flag to disable printing.

Reply:
though let's not solve it here :)

Reply:
The issue here is that we're not following https://docs.scrapy.org/en/latest/topics/practices.html#run-scrapy-from-a-script in the tests because it's problematic. So, it may be problematic for our users as well.

Reply:
I agree!